data/json/items/melee/unarmed_weapons.json

```
@@ -133,6 +133,36 @@
     "qualities": [ [ "CUT", 1 ], [ "BUTCHER", 12 ] ],
     "flags": [ "UNARMED_WEAPON", "DURABLE_MELEE", "SHEATH_KNIFE" ]
   },
+  {
+    "id": "pierce_katar",
+    "type": "GENERIC",
+    "name": { "str": "jamadhar" },
+    "copy-from": "knuckle_katar",
+    "description": "A steel brace with two thinner, pointed blades at the end, designed to primarely stab victims when used to punch.",
+    "material": [ "steel" ],
+    "volume": "600 ml",
+    "cutting": 18,
+    "price": "500 USD",
+    "qualities": [ [ "CUT", 1 ], [ "BUTCHER", 8 ] ],
+    "flags": [ "UNARMED_WEAPON", "DURABLE_MELEE", "SHEATH_KNIFE", "STAB" ]
+  },
+  {
+    "id": "impact_knuckle",
+    "type": "GENERIC",
+    "category": "weapons",
+    "weapon_category": [ "FIST_WEAPONS" ],
+    "symbol": "]",
+    "color": "dark_gray",
+    "name": { "str": "pair of impact knuckles", "str_pl": "pairs of impact knuckles" },
+    "description": "A pair of sturdy steel bracings, inlaid with padding material, which are then covered by hardened steel plates. Can be used to safely deliver punches full force without the risk of harming ones hands.",
```

Absolutely mental that the check failure prints 8 bloody times for this one slip-up.
```suggestion
    "description": "A pair of sturdy steel bracings, inlaid with padding material, which are then covered by hardened steel plates.  Can be used to safely deliver punches full force without the risk of harming ones hands.",
```